packages/core/src/graphql/middleware.test.ts

```diff
@@ -48,7 +39,10 @@ test("middleware serves request", async (context) => {
     bigint: 0n,
   });
 
-  const app = new Hono().use(contextMiddleware).use("/graphql", graphql());
+  const app = new Hono().use(
+    "/graphql",
+    graphql({ schema, db: database.drizzle }),
```

Comment:
Should this be the readonly drizzle? Or is that in a different PR 😅

Reply:
different pr